addons/website_forum/static/src/scss/website_forum.scss

```diff
@@ -1,42 +1,75 @@
 $gold: #eca801;
 $silver: #cccccc;
 $bronze: #eea91e;
+$o-wf-main-wrapper-width: map-get($container-max-widths, xl);
+$o-wf-main-wrapper-narrow-width: map-get($container-max-widths, md);
+$o-wf-sidebar-width: 16em !important;
+
 
 .website_forum {
-    .o_forum_ask_btn {
-        @include media-breakpoint-up(md) {
-            box-shadow: $box-shadow;
-            width: 200px;
-        }
-    }
+    --o-main-wrapper-width: #{$o-wf-main-wrapper-narrow-width};
+    --o-sidebar-width: #{$o-wf-sidebar-width};
 
-    .o_wforum_nav .nav-link {
-        @include o-hover-text-color($body-color, $link-color);
-        line-height: 1;
+    @include media-breakpoint-up(md){
+        --o-main-wrapper-width: #{$o-wf-main-wrapper-width};
+    }
 
-        .fa {
-            opacity: 0.5;
+    .o_wforum_main_wrapper {
+        @include media-breakpoint-up(md){
+            grid-template-columns: 1fr minmax(0, var(--o-main-wrapper-width)) 1fr;
+            gap: 1.2rem;
         }
+    }
 
-        &:hover, &.active {
-            .fa {
-                opacity: 1;
+    .o_wforum_sidebar {
+        width: var(--o-sidebar-width);
+        grid-template-rows: auto min-content;
+
+        .o_wforum_sidebar_section .nav-link{
+            border-radius: $border-radius;
+            &:hover {
+                background-color: $gray-200;
             }
         }
 
-        &.active {
-            background-color: rgba(map-get($theme-colors, 'info'), 0.1);
-            color: darken(map-get($theme-colors, 'info'), 15%)!important;
+        img.o_forum_avatar {
+            width: 50px;
+            height: 50px;
         }
+    }
 
+    .o_wforum_table {
         img.o_forum_avatar {
-            width: 30px;
-            height: 30px;
+            width: 36px;
+            height: 36px;
+            border: 2px solid var(--body-bg);
+        }
+
+        .o_wforum_reply_count, .o_wforum_view_count, .o_wforum_last_activity {
+            width: 100px;
+        }
+
+        tbody tr:hover {
+            background-color: darken($body-bg, 2%);
```

Comment:
The background is `color-4` and we don't know if it will be light or dark.... it would be better to adjust this condition accordingly ;)